lib/square.go

```diff
@@ -0,0 +1,17 @@
+package lib
+
+import (
+	"math"
+	"time"
+)
+
+type squareLatencySummand struct {
+	amplitude time.Duration
+	period    time.Duration
+}
+
+func (s squareLatencySummand) getLatency(elapsed time.Duration) time.Duration {
+	return time.Duration(
+		(2 * (2 * math.Floor((1 / float64 (s.period)) * float64(elapsed)) - math.Floor(2 * (1 / float64 (s.period)) * float64(elapsed))) + 1) * float64(s.amplitude),
+	)
```

Comment:
Given that duration is actually an int under the hood and that integer division floors the result so long that it is non-negative, I think that it is possible to implement this formula without casting anything to float. Would you mind refactoring it?